lib/checkPullRequestJob.js

```diff
@@ -0,0 +1,58 @@
+const {SERVER_JOBS_ADMIN} = require('../userWhitelist.json');
+/**
+ * @param {string} filename
+ */
+const isJobFile = (filename) => {
+  const REGISTRY_FILENAME = 'jobs_registry.py';
+  const JOB_FILE_FIX = '_jobs_';
```

Comment:
What does FIX represent here?

Reply:
I wasn't sure of what to name since this isn't a prefix or suffix.